xwiki-commons-core/xwiki-commons-xml/src/main/java/org/xwiki/xml/internal/html/DefaultHTMLCleaner.java

```diff
@@ -283,6 +286,16 @@ private CleanerTransformations getDefaultCleanerTransformations(HTMLCleanerConfi
         tt.addAttributeTransformation(HTMLConstants.ATTRIBUTE_STYLE, "text-align:center");
         defaultTransformations.addTransformation(tt);
 
+        if (getHTMLVersion(configuration) == 5) {
+            // Font tags are removed before the filters are applied in HTML5, we thus need a transformation here.
+            defaultTransformations.addTransformation(new FontTagTransformation());
+
+            tt = new TrimAttributeTagTransformation(HTMLConstants.TAG_TT,
+                HTMLConstants.TAG_SPAN);
+            tt.addAttributeTransformation(HTMLConstants.ATTRIBUTE_CLASS, "${class} monospace");
```

Comment:
Monospace is some kind of standard CSS class? Isn't it a font-family style?

Reply:
``monospace`` CSS class is already used when rendering ``##foo## `` wiki syntax to HTML5, so @michitux is simply being consistent.